NextExample/next.config.js

```diff
@@ -1,14 +1,25 @@
 const { withExpo } = require('@expo/next-adapter');
 
+const withPlugins = require('next-compose-plugins');
+
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === '1',
```

Comment:
I'm afraid `ANALYZE` is not informative enough, can we rename it to `ANALYZE_BUNDLE` or `ENABLE_BUNDLE_ANALYZER`?